src/code/z_play.c

```diff
@@ -43,13 +43,14 @@ void func_800BC590(GlobalContext* globalCtx) {
     }
 }
 
-void func_800BC5E0(GlobalContext* globalCtx, s32 transitionType) {
+void Gameplay_SetupTransition(GlobalContext* globalCtx, s32 transitionType) {
     TransitionContext* transitionCtx = &globalCtx->transitionCtx;
 
     bzero(transitionCtx, sizeof(TransitionContext));
 
     transitionCtx->transitionType = transitionType;
 
+    // types above 32 are treated as circles
     if ((transitionCtx->transitionType >> 5) == 1) {
```

Comment:
I think this comment can be confusing, as the conditional isn't checking `>= 32`. It relies on knowing that the circle types 32-55 are the only types past 32 so idk it's weird

I would just remove the comment, only indicating it's checking for circle, like

```suggestion
    if ((transitionCtx->transitionType >> 5) == 1) { // circle types
```